### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by catarinareis98m**
Part of the support team reviewing the documentation

### CODE REVIEW

File: SharePoint/SharePointOnline/manage-query-suggestions.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Automatic query suggestions are generated periodically for each result source an
 For example, in the following screenshot, "contoso" is automatically suggested.
 ![Screenshot of Query Suggestion.](media/query-suggestion.png)
   
-SharePoint has both a classic and a modern search experience, [learn about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md). The modern search experience uses the same default result source as the classic search experience. Automatic query suggestions for the default result source appear in both the classic and modern search experiences.
+SharePoint has both a classic and a modern search experience, [learn about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md). The modern search experience uses the same default result source as the classic search experience. Automatic query suggestions for the default result source appear only in the classic search experience.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-query-suggestions.md:31]:
```suggestion
SharePoint has both a classic and a modern search experience, [learn about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md). The modern search experience uses the same default result source as the classic search experience. Automatic query suggestions for the default result source appear only in the classic search experience.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @catarinareis98m, kindly commit the minor additional change. Many thanks.

**Comment by wobba** [comment]
@scanum does closing mean the change will be manually part of another branch update?

**Comment by scanum** [comment]
Hi @wobba Hope all is doing well. Could youn please review the copy review and confirm the changes if you approve?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by wobba** [comment]
Looks good to me 👍

**Comment by scanum** [comment]
@SerdarSoysal 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**